components/index.ts

```diff
@@ -376,6 +319,62 @@ app.use(
         }),
 )
 
+app.get(
```

Comment:
I really hope this doesn't break scpc

Reply:
Correction: it **FIXES** it. Someone broke it.